Adds documentation for debian architecture variant support (#4890)
theforeman/foreman-documentation: guides/common/modules/proc_adding-deb-repositories-by-using-web-ui.adoc

====
Ensure that you enter both *Releases* and *Components* exactly as they are in an `/etc/apt/sources.list` file.
====

> . Optional: In the *Architectures* field, enter one or multiple architectures separated by whitespace.

Reviewer: I don't know what the field looks like, but I would be more specific about the kind of "whitespace".

```suggestion
. Optional: In the *Architectures* field, enter one or multiple architectures separated with spaces.
```

Reviewer 2: Applied.

+

> {Project} always synchronizes packages of architecture `all` regardless of your architecture selection.

Reviewer: I'm not sure what this meant. Feel free to ignore if this would change the meaning.

```suggestion
{Project} always synchronizes packages for all architectures regardless of your architecture selection.
```

Reviewer 2: No change: the architecture is called "all". Example: https://oss.atix.de/Debian13/pool/
